debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerDefaultValueConverter.java

```diff
@@ -118,19 +118,19 @@ private Object convertDefaultValue(Object defaultValue, Column column) {
         final Map<String, DefaultValueMapper> result = new HashMap<>();
 
         // Exact numbers
-        result.put("bigint", v -> Long.parseLong(v.substring(2, v.length() - 3))); // Sample value: ((3147483648.))
-        result.put("int", v -> Integer.parseInt(v.substring(2, v.length() - 2))); // Sample value: ((2147483647))
-        result.put("smallint", v -> Short.parseShort(v.substring(2, v.length() - 2))); // Sample value: ((32767))
-        result.put("tinyint", v -> Short.parseShort(v.substring(2, v.length() - 2))); // Sample value: ((255))
+        result.put("bigint", v -> nullableDefaultValueMapper(v, value -> Long.parseLong(value.replaceAll("(\\.)$", "")))); // Sample value: ((3147483648.))
```

Comment:
Using an inline regexp isn't ideal for this one. At the very least, this should be pre-compiled into a `Pattern`, but actually, there's a better way:

    Long.parseLong(value.charAt(lastIndex) == '.' ? value.substring(0, lastIndex) : value)

It's an order of magnitude faster as per a quick JMH benchmark:

```
Benchmark                                  Mode  Cnt          Score          Error  Units
TrimBenchmark.testCompiledRegexp          thrpt    3   46507099,171 ±  3115790,518  ops/s
TrimBenchmark.testRegexp                  thrpt    3   24495251,001 ± 18640458,291  ops/s
TrimBenchmark.testSubstringEndsWith       thrpt    3  286420819,975 ± 38003048,105  ops/s
TrimBenchmark.testSubstringCharAt         thrpt    3  301351671,658 ± 86578981,123  ops/s
```